cmd/action/ci/ci.go

```diff
@@ -63,3 +65,33 @@ func setCfgName(cfg *v1alpha1.Environment) {
 	}
 	cfg.Name = fmt.Sprintf("ci%s-%s", attempt, sha)
 }
+
+// instanceTags returns the tags to be applied to the EC2 instance
+// based on the GitHub environment variables https://docs.github.com/en/actions/learn-github-actions/variables
+func instanceTags() []types.Tag {
+	job := os.Getenv("GITHUB_JOB")
+	repository := os.Getenv("GITHUB_REPOSITORY")
+	actor := os.Getenv("GITHUB_ACTOR")
+	sha := os.Getenv("GITHUB_SHA")
+
+	tags := []types.Tag{
+		{
+			Key:   aws.String("GITHUB_JOB"),
+			Value: aws.String(job),
+		},
+		{
+			Key:   aws.String("GITHUB_REPOSITORY"),
+			Value: aws.String(repository),
+		},
+		{
+			Key:   aws.String("GITHUB_ACTOR"),
+			Value: aws.String(actor),
+		},
+		{
+			Key:   aws.String("GITHUB_SHA"),
+			Value: aws.String(sha),
+		},
+	}
+
+	return tags
```

Comment:
What about:
```suggestion
	envs := []string{
		"GITHUB_JOB",
		"GITHUB_REPOSITORY",
		"GITHUB_ACTOR",
		"GITHUB_SHA",
	}

	var tags []types.Tag
	for _, env := range envs {
		tag := types.Tag{
			Key: aws.String(env)
			Value: os.Getenv(env)
		}
		tags = append(tags, tag)
	}
	return tags
```
also, what should we do if an envvar is not set?

Reply:
this will only run on the CI mode, and those ENV VAR's are guaranteed by GitHub to always be set